CMakeLists.txt

```
@@ -102,6 +102,12 @@ elseif(ANDROID)
     target_compile_definitions(platform_wsi INTERFACE VK_USE_PLATFORM_ANDROID_KHR)
 elseif(APPLE)
     target_compile_definitions(platform_wsi INTERFACE VK_USE_PLATFORM_MACOS_MVK VK_USE_PLATFORM_METAL_EXT)
+    
+# Build for iOS and tvOS
+    option(APPLE_EMBEDDED "Apple non desktop devices" OFF)
+    if (APPLE_EMBEDDED)
+        add_definitions(-D__APPLE_EMBEDDED__)
+    endif()
```

This should be moved out of the platform WSI stuff and done further down with the other `loader_common_options` stuff.
```suggestion
if(APPLE)
    # Build for iOS and tvOS
    option(APPLE_EMBEDDED "Apple non desktop devices" OFF)
    if (APPLE_EMBEDDED)
        target_compile_definitions(loader_common_options PRIVATE __APPLE_EMBEDDED__)
    endif()
endif()
```

---

Done.